tests/test_line.py

```diff
@@ -272,3 +272,32 @@ def test_from_dict_current():
     assert d1.length == 4
```

Comment:
Somewhere we should also test that the behaviour of the knobs (tracker.vars) is preserved, as done in this test:
https://github.com/xsuite/xtrack/blob/c39c22728e43a582f4405e7220ebcb381399f1af/tests/test_vs_madx.py#L455

Also it would be good to add a test for the tracker with all speedup modifications (no rad, simple quads, etc.)